features/api/job.feature

```diff
@@ -16,23 +16,65 @@ Feature: Jobs management
 
     #TODO
 
-#  Scenario: It should be possible to get all the enabled jobs.
+#START -----Filter validation-----
+
+  Scenario: It should be possible to get all the enabled jobs.
+   When I send a GET request to "/api/jobs?filter[order][enable]=true"
+  Then the response status code should be 200
+  And I should get a paged collection with the context "/api/contexts/Job"
+  And the JSON node "hydra:totalItems" should be equal to #TODO
+  And the job should have a enabled with the value "true"
+      #TODO
+
+  Scenario: It should be possible to order jobs by ID, title or abbreviation.
+  When I send a GET request to "/api/jobs?filter[order][id]"
+  Then the response status code should be 200
+  When I send a GET request to "/api/jobs?filter[order][abbreviation]"
+  Then the response status code should be 200
+  When I send a GET request to "/api/jobs?filter[order][title]"
+  Then the response status code should be 200
     #TODO
 
-#  Scenario: It should be possible to order jobs by ID, title or abbreviation.
+  Scenario: It should be possible to find a job by its ID or title.
+  When I send a GET request to "/api/jobs?filter[where][title]=User Experience Connector"
+  Then the response status code should be 200
+  And I should get a paged collection with the context "/api/contexts/Job"
+  And the JSON node "hydra:totalItems" should be equal to #TODO
+  And the job should have a title with the value "User Experience Connector"
     #TODO
 
-#  Scenario: It should be possible to find a job by its ID or title.
+  Scenario: It shoud be possible to find jobs by their abbreviation (an abbreviation may have several jobs).
+  When I send a GET request to "/api/jobs?filter[where][abbreviation]=DEPT"
+  Then the response status code should be 200
+  And I should get a paged collection with the context "/api/contexts/Job"
+  And the JSON node "hydra:totalItems" should be equal to #TODO
+  And the job should have an abbreviation with the value "DEPT"
     #TODO
 
-#  Scenario: It shoud be possible to find jobs by their abbreviation (an abbreviation may have several jobs).
+#END -----Filter validation-----
+
+#START -----Crud validation-----
+
+  Scenario: It should be possible to create a new job
+  When I send a POST request to "/api/jobs" with body:
+    | | |
     #TODO
 
-#  Scenario: When creating a new job, it must have at least one mandate. By default is for the ongoing mandate.
+  Scenario: When creating a new job, it must have at least one mandate. By default is for the ongoing mandate.
```

Comment:
To be removed as it's not true. Later we will have prospects, clients or even alumni who won't have a job bound to a mandate.